applications/blogs/pom.xml

```diff
@@ -155,36 +104,6 @@
                 </pluginManagement>
             </build>
         </profile>
-        <profile>
-            <id>autoInstallBundleEnvironment</id>
-            <activation>
-                <activeByDefault>false</activeByDefault>
-            </activation>
-            <build>
-                <pluginManagement>
-                    <plugins>
-                        <plugin>
-                            <groupId>org.apache.sling</groupId>
-                            <artifactId>sling-maven-plugin</artifactId>
-                            <executions>
-                                <execution>
-                                    <id>install-bundle</id>
-                                    <goals>
-                                        <goal>install</goal>
-                                    </goals>
-                                    <configuration>
-                                        <slingUrl>${env.WEBSIGHT_URL}:${env.WEBSIGHT_PORT}/system/console
-                                        </slingUrl>
-                                        <user>${env.WEBSIGHT_ADMIN_USER}</user>
-                                        <password>${env.WEBSIGHT_ADMIN_PASSWORD}</password>
-                                    </configuration>
-                                </execution>
-                            </executions>
-                        </plugin>
-                    </plugins>
-                </pluginManagement>
-            </build>
-        </profile>
     </profiles>
 
     <dependencyManagement>
```

Comment:
We have a dependencyManagement in the applications module and here is a copy of it.

Reply:
Same with the pluginManagement. And it's also in the common module. I think we could remove those sections.